hyperframe/frame.py

```diff
@@ -110,6 +110,14 @@ def parse_frame_header(header):
         stream_id = fields[4]
 
         if type not in FRAMES:
+            if not strict:
```

Comment:
This is purely preference, but I’d have done this as:

```python
if strict:
    raise UnknownFrameError
return ExtensionFrame
```

because we expect `strict=True` to be the unusual path – or at least, it seems so, given we default to `False`.

(I don‘t care if you change this or not, just noting my feelings.)